src/core/server/ui_settings/saved_objects/migrations_790.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { SavedObjectSanitizedUiSettings, SavedObjectUnsanitizedUiSettings } from './types';
+
+export const migrations790 = {
+  '7.9.0': (doc: SavedObjectUnsanitizedUiSettings<any>): SavedObjectSanitizedUiSettings<any> => ({
+    ...doc,
+    ...(doc &&
+      doc.attributes && {
+        attributes: Object.keys(doc.attributes).reduce(
+          (acc, key) =>
+            key.indexOf('siem') > -1
```

Comment:
NIT: use `startsWith` instead

Overall, taking the previous comments into account, I think the whole block can be changed to

```ts
{
  ...doc,
  attributes: Object.keys(doc.attributes).reduce(
    (acc, key) =>
      key.startsWith('siem:')
        ? { ...acc, [key.replace('siem', 'securitySolution')]: doc.attributes[key] }
        : acc,
    {}
  ),
}
```

Reply:
need a check on doc.attributes existence but i can do the startsWith